frontend/package.json

```diff
@@ -23,16 +24,15 @@
     "react-dom": "17.0.2",
     "react-dropzone": "12.1.0",
     "react-router-dom": "6.3.0",
-    "react-scripts": "5.0.1",
     "react-tsparticles": "1.43.1",
     "uuid": "8.3.2",
+    "vite": "^3.0.4",
+    "vite-tsconfig-paths": "^3.5.0",
```

Comment:
Are these required during runtime or can they be moved to devDependencies?

Reply:
good point, these don't need to be here